ontopy/ontology.py

```diff
@@ -2025,7 +2061,7 @@ def get_wu_palmer_measure(self, cls1, cls2):
         generations2 = self.number_of_generations(cls2, cca)
         return 2 * ccadepth / (generations1 + generations2 + 2 * ccadepth)
 
-    def new_entity(
+    def new_entity(  # pylint: disable=too-many-arguments,too-many-branches,too-many-positional-arguments
```

Comment:
These pylint disable's will probably make the line too long. Probably better to move the comment to after the docstring (line 2119)